sample-queries/sample-queries.json

```diff
@@ -1432,6 +1432,16 @@
         "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/api/channel_get_messagereply",
         "tip": "This query requires a group id of the Team, channel id of the corresponding channel of that Team, message id of the message of which you need the reply and the id of the specific reply. To find the group id, channel id, message-id and reply-id you can run: 1) GET https://graph.microsoft.com/beta/me/joinedTeams 2) GET https://graph.microsoft.com/beta/groups/{group-id-for-teams}/channels 3) GET https://graph.microsoft.com/beta/teams/{group-id-for-teams}/channels/{channel-id}/messages 4) GET https://graph.microsoft.com/beta/teams/{group-id-for-teams}/channels/{channel-id}/messages/{message-id}/replies",
         "skipTest": false
+    },    
+    {
+        "id": "0d0cddaf-aae2-4541-8104-1964d22a8da4",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "apps installed for user",
+        "requestUrl": "/beta/users/{user-id}/teamwork/installedApps?$expand=teamsApp",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/userteamwork-list-installedapps?view=graph-rest-beta&tabs=http",
+        "tip": "This query requires a user id. To find user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
```

Comment:
There is an extra space here between `run:  GET`

Reply:
7ccf64d8fc2d8068f979e11c5f8290bf2e58f3de